addons/website/views/snippets/s_features_wave.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_features_wave" name="Features Wave">
+    <section class="s_features_wave o_cc o_cc5 pt64 pb64" data-oe-shape-data="{'shape':'web_editor/Wavy/11', 'showOnMobile':true}">
+        <div class="o_we_shape o_web_editor_Wavy_11 o_shape_show_mobile"/>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 pb-24">
+                    <h3 style="text-align: center;">Everything you need</h3>
+                    <p class="lead" style="text-align: center;">List and describe the key features of your solution or service.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-thumbs-up d-block mx-auto rounded fs-5 bg-o-color-1"/>
+                    <br/>
+                    <h5 style="text-align: center;">Reliability</h5>
+                    <p class="o_small" style="text-align: center;">Consistent performance and uptime ensure efficient, reliable service with minimal interruptions and quick response times.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-trophy d-block mx-auto rounded fs-5 bg-o-color-1"/>
```

Comment:
We prefer to use `fa-[x]-o` icons since they typically offer a cleaner visual result. However, there aren't many available, so feel free to reuse the ones used in the other new `s_features_[x]` snippets  👍